test/security/CookieSecurity.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+describe('CookieSecurity', function() {
+  var CookieSecurity = require('../../').CookieSecurity;
+  var cookie = "cookie-value";
+  var headers = {
+    'set-cookie': [
+      cookie
+    ]
+  };
+
+  it('is a function', function() {
+    CookieSecurity.should.be.type('function');
+  });
+
+  describe('cookie value', function () {
+    var instance1 = new CookieSecurity(cookie);
+    var instance2 = new CookieSecurity(headers);
+
+    it('is accepted with cookie string', function () {
+      instance1.should.have.property("cookie", cookie);
+    });
+
+    it('is accepted with last header', function () {
+      instance2.should.have.property("cookie", cookie);
+    });
+  });
+
+  describe('defaultOption param', function() {
+    it('is accepted as the second param', function() {
+      new CookieSecurity(cookie, {});
```

Comment:
I just copied and modified the existing BearerSecurity test. I'm not sure what is missing since nether bearer or cookie  security tests look any different.